variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "cluster_enabled_log_types" {
```

Comment:
Since you are adding a new variable, you need to update the README: https://github.com/terraform-aws-modules/terraform-aws-eks#doc-generation

Reply:
Ah yes, will do.